test_html/test_see_details.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
```

Comment:
@pdpinch Would there be any problem using this for test data regarding copyright or license?

Reply:
I'll replace most of the text with random latin